include/grpcpp/security/credentials.h

```diff
@@ -98,10 +98,10 @@ class ChannelCredentials : private GrpcLibraryCodegen {
   // This function should have been a pure virtual function, but it is
   // implemented as a virtual function so that it does not break API.
   virtual std::shared_ptr<Channel> CreateChannelWithInterceptors(
-      const grpc::string& target, const ChannelArguments& args,
+      const grpc::string& /* target */, const ChannelArguments& /* args */,
```

Comment:
This looks odd. Is this the normal way to do this?

Reply:
Like why not just remove the names altogether if this is a thing?

Reply:
Internal CL number - 244876205
I will leave it to @justsomeguy-google-com